### PR TITLE
docs: update keycloak.md

### PR DESCRIPTION
**Pull request by abdaziz**
The documentation is excellent, but after following it, i'v experienced an issue 401, and the UI loop to login, to fix it i restarted the argocd-server pod then it' worked well.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] The title of the PR conforms to the [Toolchain Guide](https://argo-cd.readthedocs.io/en/latest/developer-guide/toolchain-guide/#title-of-the-pr)
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [x] Does this PR require documentation updates?
* [x] I've updated documentation as required by this PR.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md#legal)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).
* [ ] My new feature complies with the [feature status](https://github.com/argoproj/argoproj/blob/master/community/feature-status.md) guidelines.
* [ ] I have added a brief description of why this PR is necessary and/or what this PR solves.
* [ ] Optional. My organization is added to USERS.md.
* [ ] Optional. For bug fixes, I've indicated what older releases this fix should be cherry-picked into (this may or may not happen depending on risk/complexity).

### CODE REVIEW

**Comment by abdaziz** [comment]
argocd integration with keycloak, edit docs and add rolling restart of argocd-server at the end.

File: docs/operator-manual/user-management/keycloak.md [hunk 1]
```diff
@@ -125,3 +125,7 @@ In this example we give the role _role:admin_ to all users in the group _ArgoCDA
 You can now login using our new Keycloak OIDC authentication:
 
 ![Keycloak ArgoCD login](../../assets/keycloak-login.png "Keycloak ArgoCD login")
+
+## Trouleshoot
```
Comment (wanghong230) [inline docs/operator-manual/user-management/keycloak.md:129]:
```suggestion
## Troubleshoot

```

File: docs/operator-manual/user-management/keycloak.md [hunk 2]
```diff
@@ -125,3 +125,7 @@ In this example we give the role _role:admin_ to all users in the group _ArgoCDA
...
+if ArgoCD auth returns 401 or when the login attempt leads to the loop, then restart the argocd-server pod.
```
Comment (wanghong230) [inline docs/operator-manual/user-management/keycloak.md:130]:
```suggestion
If ArgoCD auth returns 401 or when the login attempt leads to the loop, then restart the argocd-server pod.
```

File: docs/operator-manual/user-management/keycloak.md [hunk 3]
```diff
@@ -125,3 +125,7 @@ In this example we give the role _role:admin_ to all users in the group _ArgoCDA
...
+kubectl rollout restart deployment argocd-server -n argocd
```
Comment (wanghong230) [inline docs/operator-manual/user-management/keycloak.md:131]:
```suggestion

`kubectl rollout restart deployment argocd-server -n argocd `
```

**Comment by abdaziz** [comment]
Hi Wanghong,

Thank yoy for the review and correction.

Regard.

**Comment by christianh814** [comment]
@abdaziz This looks good, can you update your PR with the requested changes from @wanghong230?

**Review by wanghong230 — APPROVED** [review]

**Review by pasha-codefresh — APPROVED** [review]

**Pull request merged**